go/src/meguca/parser/commands.go

```diff
@@ -49,6 +49,20 @@ func parseCommand(match []byte, board string) (com common.Command, err error) {
 			com.Eightball = answers[randInt(len(answers))]
 		}
 
+	// Increment pyu counter
+	case bytes.Equal(match, []byte("pyu")):
+		if config.GetBoardConfigs(board).Pyu {
```

Comment:
Returns empty struct, if config set to false.

Reply:
I'm not sure what to do about this one, it's directly related to `AssertDeepEquals` in `commands_test.go`. I've tried a lot of things.

Reply:
Probably best to define a specific error in the global scope `var errPyuDisabled = errors.New("pyu disabled")` and then check for this error in the calling function. If matched, skip command.

Reply:
What do you mean?

Reply:
I'm not really sure. I can't get a lot of information from what's going on in this test.
What's weird is the fact that things work even though the test fails.